docs/guides/immutable-configurations/immutable-configuration.md

```diff
@@ -524,27 +695,46 @@ export MY_APP_REDIRECT_URIS_1=https://example.com/logout
 
 ### Configuration File Not Loaded
 
-**Symptom:** Application not found after startup.
+**Symptom:** Application or flow graph not found after startup.
 
 **Cause:** File not in correct directory or invalid YAML.
 
 **Solution:**
-1. Verify file location: `repository/resources/applications/`
-2. Check YAML syntax: `yamllint my-app.yaml`
+1. Verify file location:
+   - Applications: `repository/resources/applications/`
+   - Flow graphs: `repository/resources/graphs/`
+   - Identity providers: `repository/resources/identity_providers/`
+2. Check YAML syntax: `yamllint my-config.yaml`
 3. Check server logs for parsing errors
+4. Ensure the `id` and `handle` fields are unique
 
-### Cannot Create Applications
+### Cannot Create Applications or Flow Graphs
 
-**Symptom:** POST /applications returns error.
+**Symptom:** POST /applications or POST /flow/graphs returns error.
 
 **Cause:** Immutable mode is enabled.
 
 **Solution:**
-This is expected behavior. To add new applications:
-1. Create a new YAML file in `repository/resources/applications/`
+This is expected behavior. To add new resources:
+1. Create a new YAML file in the appropriate directory:
+   - Applications: `repository/resources/applications/`
+   - Flow graphs: `repository/resources/graphs/`
```

Comment:
```suggestion
   - Flow graphs: `repository/resources/flow/`
```

can we use the path as /flow? do we need to use the word graphs ? In the API also we are using the term flow